samples/Mvc.Formatters/Mvc.Formatters/Views/Home/Index.cshtml

```diff
@@ -0,0 +1,36 @@
+﻿@{
+    ViewData["Title"] = "Posting data in text/plain format";
+}
+<h2>Posting data in text/plain format</h2>
+<div>
+    <label>Sample text:</label>
+    <input type="text" id="sampleText" />
+    <button id="submitText" onclick="javascript: submitData()">Submit</button>
+    <div id="divStatus" style="margin-top:20px">
+        <label style="font-weight:bold">Echoed value: </label>
+        <label id="lblStatus"></label>
+    </div>
+</div>
+@section Scripts{
+    <script src="https://code.jquery.com/jquery-3.1.0.min.js"></script>
+    <script type="text/javascript">
```

Comment:
Can we do this with a regular form post instead of using JavaScript? It doesn't add value and just distracts customers from the important part of the sample.